python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1623,7 +1634,11 @@ def _filterArtifacts(self, spanSetList, epochCountImage):
         for i, span in enumerate(spanSetList):
             y, x = span.indices()
             counts = epochCountImage.array[[y1 - y0 for y1 in y], [x1 - x0 for x1 in x]]
-            nCountsBelowThreshold = numpy.count_nonzero((counts > 0) & (counts <= self.config.maxNumEpochs))
+            totalCounts = nImage.array[[y1 - y0 for y1 in y], [x1 - x0 for x1 in x]]
+            countThreshold = min(self.config.maxNumEpochs,
+                                 int(self.config.maxNumEpochsMinFraction * numpy.mean(totalCounts)))
+            nCountsBelowThreshold = numpy.count_nonzero((counts > 0) &
+                                                        (counts <= countThreshold))
```

Comment:
Inclusive below might be confusing.

Reply:
The documentation and naming of "maxNumEpochs" implies inclusive below.

It is the maximum number of epochs an artifact can appear in and still be clipped.  Of course, ee COULD reframe the config/language to be "the minimum number of epochs a REAL SOURCE could appear an outlier and NOT be clipped", but that would require renaming existing configs. Reframing the problem is out of scope for this ticket.

Reply:
I'll temporary variable to "effectiveMaxNumEpochs"